app/views/admin/action_items/_rejected_auctions.html.erb

```diff
@@ -15,7 +15,7 @@
       <td><%= link_to(auction.title, admin_auction_path(auction.id)) %></td>
       <td><%= auction.delivery_due_at %></td>
       <td><%= auction.delivery_url %></td>
-      <td><%= link_to auction.winning_bidder_name, admin_user_path(auction.winning_bidder_id) %></td>
+      <td><% if auction.winning_bidder? %><%= link_to auction.winning_bidder_name, admin_user_path(auction.winning_bidder_id) %><% end %></td>
```

Comment:
why is calling `html_safe` on something from the presenter bad? This situation feels similar to the other situations where we branch depending on `Guest` or `User` so using the same solution sounds alright to me...